components/formats-bsd/src/loci/formats/in/JPEGReader.java

```diff
@@ -207,8 +211,28 @@ protected void initFile(String id) throws FormatException, IOException {
         }
 
         HashMap<String, String> tags = exif.getTags();
-        for (String tagName : tags.keySet()) {
-          addGlobalMeta(tagName, tags.get(tagName));
+        for (String tagName : tags.keySet()) {  
+          if (tagName.equals(exif.getTagName(ExifSubIFDDirectory.TAG_X_RESOLUTION))) {
+            try {
+              Length sizeX = FormatTools.getPhysicalSizeX(Double.parseDouble(tags.get(tagName)));
```

Comment:
As mentioned in a conversation, my reading of the EXIF spec (see e.g http://www.exiv2.org/tags.html) is that the `{XY}Resolution` value should be map to the inverse of our `PhysicalSize{XY}`.